src/_includes/curated_publications.html

```diff
@@ -4,6 +4,22 @@ <h2 class="section-intro__title">{{include.title}}</h2>
     <p class="section-intro__description">{{include.description}}</p>
   </header>
 
+  {% assign posts = site.posts | where: 'pinned_locations', include.pinned_location %}
+  {% for post in posts limit: 1 %}
+    {% if site.lang == "es" %}
+      {% assign title = post.title-es %}
+      {% assign description = post.description-es %}
+    {% else %}
+      {% assign title = post.title %}
+      {% assign description = post.description %}
+    {% endif %}
+    {% assign content_type = site.translations[site.lang]publications.content-type %}
+    {% assign cta = site.translations[site.lang]publications.teaser-cta %}
+    {% assign cta-href = post.url | prepend: site.baseurl %}
+    {% assign image_src = post.image.src | prepend: site.baseurl %}
+    {% include hero_teaser.html class="curated-publications__hero" content_type=content_type cta=cta cta-href=cta-href description=description image_position="right" image_src=image_src title=title %}
```

Comment:
Not one for this pull request because **we should release this feature first** but let's rename some of these variables and attributes for more consistent casing. We've got a mix of kebab and python case. 

Liquid seems to prefer python case: https://shopify.github.io/liquid/tags/variable/